gpkit/solution_array.py

```diff
@@ -6,6 +6,53 @@
 from .small_scripts import unitstr, mag
 
 
+def senss_table(data, showvars=(), title="Sensitivities", minval=1e-2,
+                **kwargs):
+    "Returns sensitivity table lines"
+    if "constants" in data.get("sensitivities", {}):
+        data = data["sensitivities"]["constants"]
+    if showvars:
+        data = {k: data[k] for k in showvars if k in data}
+    return results_table(data, title, minval=minval, sortbyvals=True,
+                         valfmt="%+-.2g ", vecfmt="%+-8.2g",
+                         printunits=False, **kwargs)
```

Comment:
now that we have the summary table I propose that `minval` should default to zero (a user printing the whole table probably wants to see everything).